_i18n/en.yml

```diff
@@ -501,6 +501,11 @@ faq:
   avuln: >
     The Monero community has created a series of videos called "Breaking Monero", where potential Monero vulnerabilities are explored and discussed. There are 14 videos, with each exploring a different subject. Check out <a href="https://www.monerooutreach.org/breaking-monero/">the videos on monerooutreach.org</a>.
   vulnspotify: Available on Spotify as podcast
+  qasicresistance: "What is ASIC resistance? Why is it important?"
+  aasicresistance: ASICs are basically special computers created to do only one job, contrary to normal computers, which are made for multiple purposes. This characteristic makes ASICs very efficient for @mining.
+  aasicresistance1: The problem is that these devices are very expensive, consume a lot of electricity and can be afforded by few, wealthy people. This leads to few entities owning a big amount of the hashrate of the network, which is a serious threat to the security of the network itself. For example, if big ASIC operators collude and manage to gain the majority of the hashrate of the network, they could arbitrarily reject transactions.
```

Comment:
See my other comment in regards to energy consumption. Also I would remove the mention of "wealthy" people, it doesn't really fit in. Many miners aren't wealthy, they just hope they can recoup their costs.

The other problem with ASICs is that ASIC manufacturers often use their own devices to mine before selling them to the public, so by the time they are available to the public they are outdated and unprofitable.

Reply:
> Many miners aren't wealthy, they just hope they can recoup their costs.

My assumption was that if you can afford miners that cost thousands of dollars each, you are probably wealthy. But i see your point.